quickstart/101-virtual-network-manager-create-mesh/variables.tf

```diff
@@ -0,0 +1,11 @@
+        variable "resource_group_location" {
```

Comment:
Please run `terraform fmt` command in this folder to format the code.

Reply:
Command run.